DARPA-ASKEM/terarium
# fix for config saving issue

# Description

Issue: 
- Extracting an inputs from a document in the model config then changing an input value and saving does not save the new value

Testing:
- upload a document and model to a workflow
- create a model config and add the model & document
-  extract the inputs from the document
- make changes to a few of the inputs and save
- check that the inputs have been updated in the new config

Resolves #(issue)
